recipes/git-absorb/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -o xtrace -o nounset -o pipefail -o errexit
+
+# check licenses
+cargo-bundle-licenses \
+    --format yaml \
+    --output THIRDPARTY.yml
+
+# build statically linked binary with Rust
+cargo install --no-track --locked --root ${PREFIX} --path .
+
+mkdir -p ${PREFIX}/etc/bash_completion.d 
+mkdir -p ${PREFIX}/share/fish/vendor_completions.d 
+mkdir -p ${PREFIX}/share/zsh/site-functions
+mkdir -p ${PREFIX}/share/man/man1
+install -m 644 Documentation/git-absorb.1 ${PREFIX}/share/man/man1/git-absorb.1
+git-absorb --gen-completions bash > ${PREFIX}/etc/bash_completion.d/git-absorb
```

Comment:
Why this one goes to `etc` but the others to `share`?

Reply:
This is the standard path for bash completion files: https://unix.stackexchange.com/questions/159735/where-do-bash-completion-functions-come-from.  I'm not entirely sure why `bash` uses `etc` while the other shells use `share`, but these are all the standard directories.